apps/src/lib/node/ledger/storage/mod.rs

```diff
@@ -606,6 +616,13 @@ mod tests {
             .write(&key, types::encode(&value))
             .expect("write failed");
 
+        let nonce = nonce + 1;
+        let bytes = types::encode(&nonce);
+        storage.write(&bp_nonce_key, bytes).unwrap();
+
+        let bytes = types::encode(&Uint::default());
+        storage.write(&bp_nonce_key, bytes).unwrap();
```

Comment:
Do we need this second write since we've just written the same key?

Reply:
thanks, it's not needed